core/domain/exp_domain.py

```diff
@@ -620,11 +623,13 @@ def __init__(
 
     @classmethod
     def create_default_exploration(
-            cls, exploration_id, title=feconf.DEFAULT_EXPLORATION_TITLE,
-            init_state_name=feconf.DEFAULT_INIT_STATE_NAME,
-            category=feconf.DEFAULT_EXPLORATION_CATEGORY,
-            objective=feconf.DEFAULT_EXPLORATION_OBJECTIVE,
-            language_code=constants.DEFAULT_LANGUAGE_CODE):
+            cls, exploration_id: str,
+            title: str=feconf.DEFAULT_EXPLORATION_TITLE,
+            init_state_name: str=feconf.DEFAULT_INIT_STATE_NAME,
+            category: str=feconf.DEFAULT_EXPLORATION_CATEGORY,
+            objective: str=feconf.DEFAULT_EXPLORATION_OBJECTIVE,
+            language_code: str=constants.DEFAULT_LANGUAGE_CODE
+            ) -> Exploration:
```

Comment:
Why do you type this file?

Reply:
because here https://github.com/oppia/oppia/blob/56a99955260f971a212fe8baf5d16a2d6bc8818b/core/domain/caching_services_test.py#L645 
and here
https://github.com/oppia/oppia/blob/56a99955260f971a212fe8baf5d16a2d6bc8818b/core/domain/caching_services_test.py#L715
we are using this function and to avoid untyped-call error i typed these functions.

Reply:
ditto add ignores in the test file for now, they will be removed later when this file is typed.

Reply:
done!